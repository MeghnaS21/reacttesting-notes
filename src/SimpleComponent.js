const SimpleComponent=()=>{
    return (
        <div>
            <h1> A simple JSX</h1>
        </div>
    )
}
export default SimpleComponent