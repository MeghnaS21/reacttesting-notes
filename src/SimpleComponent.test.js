import SimpleComponent from "./SimpleComponent";
import React from "react";
import renderer from 'react-test-renderer'

    test('matches dom snapshot',()=>{
        const domTree=renderer.create(<SimpleComponent/>).toJSON()
        expect(domTree).toMatchSnapshot()
    })