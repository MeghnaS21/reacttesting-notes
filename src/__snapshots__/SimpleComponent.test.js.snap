// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches dom snapshot 1`] = `
<div>
  <h1>
     A simple JSX
  </h1>
</div>
`;
